loleaflet/src/control/Control.JSDialogBuilder.js

```diff
@@ -1632,7 +1632,8 @@ L.Control.JSDialogBuilder = L.Control.extend({
 	},
 
 	_pushbuttonControl: function(parentContainer, data, builder, customCallback) {
-		var pushbutton = L.DomUtil.create('button', '', parentContainer);
+		var wrapper = L.DomUtil.create('div', '', parentContainer); // need for freemium overlay
```

Comment:
Perhaps call it `freemium_overlay` instead of `wrapper`, if that's the purpose?

Reply:
it was that name earlier and then I changed it because it was not making sense to me as this is an actual wrapper. Without it direct parent of the push button is actual wizard so could not add overlay as it would cover the entire wizard so this element was introduced as wrapper to accommodate both push-button and overlay.